Composer/packages/client/src/utils/runtimeErrors.ts

```diff
@@ -15,6 +15,23 @@ export const missingDotnetVersionError = {
   },
 };
 
+export const missingFunctionsError = {
+  message: formatMessage('To run this bot, Composer needs Azure Functions Core Tools.'),
+  linkAfterMessage: {
+    text: formatMessage('Learn more.'),
+    url:
+      'https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local#install-the-azure-functions-core-tools',
+  },
+  link: {
+    text: formatMessage('Install Azure Functions'),
+    url: 'https://www.npmjs.com/package/azure-functions-core-tools',
+  },
+};
+
 export const checkIfDotnetVersionMissing = (err: { message: string }) => {
   return /(Command failed: dotnet user-secrets)|(install[\w\r\s\S\t\n]*\.NET Core SDK)/.test(err.message as string);
 };
+
+export const checkIfFunctionsMissing = (err: { message: string }) => {
+  return /(Azure Functions runtime not installed.)|(spawn func ENOENT)/.test(err.message as string);
```

Comment:
`ENOENT` seems a little broad. Are we sure that all `ENOENT` errors are due to functions missing?

Reply:
I am not sure -- this is the error that comes back when the `func` command is missing or not functional.

Reply:
I think we may be OK here as it explicitly checks for `func`, which is an Azure Functions thing: https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=macos%2Ccsharp%2Cbash#create-a-local-functions-project

Reply:
That's a good point. I just read right past that!